### ISSUE TITLE
Biomes are barely generating 1.18.2

### ISSUE THREAD

**Issue by aelivare** [CorgiTaco/BYG#715]
### Minecraft Version

1.18.2

### BYG Versions

1.4

### Terrablender

1.1.0.99

### Mod Loader

Forge

### Mod Loader Version

40.0.52

### Bug Severity

Major

### What happened?

I've started several survival worlds in 1.18.2 where 95% of the biomes are vanilla and the rest are BYG biomes. The biomes are tiny when they do spawn (I used Nature's Compass to find the closest Coniferous Forest, my favourite biome, to find only two coniferous trees surrounded by different non-modded biomes, and on another world the compass couldn't even find a Coniferous Forest biome). I even tried to use a Larger Biomes world, hoping I'd find a bigger Coniferous Forest, or any large modded biome at all, but I just found myself in a seemingly endless savanna with two baobab trees next to my spawn. 

My latest.log is of a creative world I just created, with only BYG and Terrablender in the mods folder, no resourcepacks loaded(even though that won't make a difference, but still) and default world settings. I put a screenshot of an example of the size of the modded biomes surrounded by massive stretches of vanilla biomes. (I hope the screenshot works, I just dragged into the textbox, this is my first time using Github haha)

![2022-04-16_21 01 16](https://user-images.githubusercontent.com/103852028/163688109-8a123972-a4f2-4ca9-8f51-41de6194514f.png)

The modded biomes are a tiny sliver in the landscape. I'm not sure what is causing this. Maybe I'm just stupid and I didn't do something I was supposed to do? Regardless, thank you in advance!

### latest.log

[latest.log](https://github.com/AOCAWOL/BYG/files/8500497/latest.log)

**Comment by CorgiTaco** [comment]
if you want the biomes to be more prominent you must fill the biome arrays in the config respectively. We explain why vanilla biomes are so prominent here: 
![image](https://user-images.githubusercontent.com/66983020/163688514-239c5f4d-e21d-4bbe-aec7-eb7fc6e57357.png)

**Comment by aelivare** [comment]
Thank you for this explanation! I didn't know the generation in 1.18 was so vastly different from previous versions. I also figured out what the issue was. I still had the same config file I used for my 1.16.5 world in there, which I had modified to suit my tastes. I removed that old file and let the mod generate a new config file, which fixed my issue! Now the modded biomes are spawning normally again. Thank you for creating this awesome mod and have a lovely day :)
